Getting-Started.md

```diff
@@ -543,6 +543,45 @@ elasticdump --input=http://localhost:9200/git_chaoss/ --output=git_data.json --t
 ```
 elasticdump --input=http://localhost:9200/git_chaoss/ --output=git_mapping.json --type=mapping
 ```
+#### Save new changes made in Dashboard by mounting the volume in ElasticSearch [&uarr;](#how-to-)
```

Comment:
You should link this question to the `How to` index. Please check the other entries in the index, references:
- https://github.com/chaoss/grimoirelab-sirmordred/blob/master/Getting-Started.md#how-to-
- raw data: https://raw.githubusercontent.com/chaoss/grimoirelab-sirmordred/master/Getting-Started.md (look for `## How to`)